### PR TITLE
updated Index.html and style.css

### PR DESCRIPTION
**Pull request by anshumanc007**
Updated the html and css file of the portfolio section.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+bin/**
+*~.nib
+local.properties
+.classpath
+.settings/
+.loadpath
+.externalToolBuilders/
+*.launch
+.cproject
+.buildpath
```
Comment (skashish-sps) [inline .gitignore:27]: Nit: Add a new line at the end of every file.

**Comment by skashish-sps** [comment]
Also deploy it, so that you can track what changes you are doing, also we can see what you are up to

File: .gitignore (hunk 1 above)
Comment (anshumanc007) [reply .gitignore:27]: Okay. I'll do it.

Comment (anshumanc007) [reply .gitignore:27]: I have deployed it. Here is the link:-  achoudhary-sps-summer20.df.r.appspot.com

**Pull request merged**